packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -442,7 +442,7 @@ class FlutterErrorDetails extends Diagnosticable {
 
   @override
   String toStringShort() {
-    return library != null ? 'Exception Caught By $library' : 'Exception Caught';
+    return library != null ? 'Exception caught by $library' : 'Exception caught';
```

Comment:
There are already some tests that check this output but they are going on the SCREAMING CAPS VERSION of this message. If you search for EXCEPTION CAUGHT BY you should find them.

Reply:
https://github.com/flutter/flutter/blob/07735578af6864adf078a8cf5343736dbc00be8a/packages/flutter/lib/src/foundation/diagnostics.dart#L1210
It seems this converts it to upper case and it is majority of the test that is test on.
Maybe create a test specifically test toStringShort?

Reply:
I added a new test for `FlutterErrorDetails.toStringShort`.